packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -0,0 +1,275 @@
+import '../../solid-components';
+import { html } from 'lit';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+import { userEvent } from '@storybook/testing-library';
+import { waitUntil } from '@open-wc/testing-helpers';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes, parameters } = storybookDefaults('sd-flipcard');
+const { generateTemplate } = storybookTemplate('sd-flipcard');
+const { overrideArgs } = storybookHelpers('sd-flipcard');
+
+export default {
+  title: 'Components/sd-flipcard',
+  component: 'sd-flipcard',
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'front',
+      value: `<p slot="front">Lorem ipsum dolor sit amet per niente da faremmasds nonnummy dolore lorem ipsum dolor sit amet consectuer</p>`
+    },
+    {
+      type: 'slot',
+      name: 'back',
+      value: `<p slot="back">Lorem ipsum dolor sit amet per niente da faremmasds nonnummy dolore lorem ipsum dolor sit amet consectuer</p>`
+    },
+    {
+      type: 'slot',
+      name: 'mediaFront',
+      value: `<img slot='mediaFront' class="object-cover h-full w-full" src='./placeholders/images/generic.jpg' alt='Generic'/>`
+    },
+    {
+      type: 'slot',
+      name: 'mediaBack',
+      value: `<img slot='mediaBack' class="object-cover h-full w-full" src='./placeholders/images/generic.jpg' alt='Generic'/>`
+    }
+  ]),
+
+  argTypes,
+
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+/**
+ * This shows sd-flipcard in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return html`<div style="width:300px">${generateTemplate({ args })}</div>`;
+  }
+};
+
+/**
+ * The sd-flipcard can be displayed in several ways using the `frontVariant` and `backVariant` attributes. This example shows the usage `frontVariant` attribute.
+ */
+
+export const Variants = {
```

Comment:
there seem to be a bug in the primary variant, as I get it displayed 2 times 
![image](https://github.com/solid-design-system/solid/assets/48211526/3630022f-3560-4d53-9ebc-ee1b16d2dd12)

Reply:
This is an emergent bug created with the interaction of our formatter and Storybook. Since the number of options is large prettier formats it like this: 
<img width="571" alt="image" src="https://github.com/solid-design-system/solid/assets/63849626/b40c9b46-4d2b-4864-81ff-967bd5b3de97">


Storybook interprets  the empty space before the first pipe(|) as an option, hence resulting in this duplication. This is solely a Storybook issue. I currently can't find a way to skip formatting for a single line, so I would suggest we leave this as a WON'T FIX. What do you think? @karlbaumhauer @mariohamann

Reply:
any solution idea @mariohamann?
@Vahid1919 if you cant fix it, pls open a ticket and put it into the backlog.

Reply:
created the ticket here: #1175